### ISSUE TITLE
TypeError in DraggableMPTTAdmin with UUIDField primary keys

### ISSUE THREAD

**Issue by tjg1** [django-mptt/django-mptt#532]
My project uses `UUIDField` as the primary key for all its models. Registering a model containing a `UUIDField` with `DraggableMPTTAdmin` throws a `TypeError` when viewing the model tree in Django admin. Using the regular `MPTTModelAdmin` works fine.

### Traceback

```
Environment:


Request Method: GET
Request URL: http://127.0.0.1:8000/admin/catalogue/category/

Django Version: 1.10.4
Python Version: 3.5.2
Installed Applications:
['catalogue.apps.CatalogueConfig',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'mptt']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']



Traceback:

File "C:\Users\tomg\virtualenv\django\lib\site-packages\django\core\handlers\exception.py" in inner
  39.             response = get_response(request)

File "C:\Users\tomg\virtualenv\django\lib\site-packages\django\core\handlers\base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "C:\Users\tomg\virtualenv\django\lib\site-packages\django\core\handlers\base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "C:\Users\tomg\virtualenv\django\lib\site-packages\django\contrib\admin\options.py" in wrapper
  544.                 return self.admin_site.admin_view(view)(*args, **kwargs)

File "C:\Users\tomg\virtualenv\django\lib\site-packages\django\utils\decorators.py" in _wrapped_view
  149.                     response = view_func(request, *args, **kwargs)

File "C:\Users\tomg\virtualenv\django\lib\site-packages\django\views\decorators\cache.py" in _wrapped_view_func
  57.         response = view_func(request, *args, **kwargs)

File "C:\Users\tomg\virtualenv\django\lib\site-packages\django\contrib\admin\sites.py" in inner
  211.             return view(request, *args, **kwargs)

File "C:\Users\tomg\virtualenv\django\lib\site-packages\mptt\admin.py" in changelist_view
  199.                     'data-context': json.dumps(self._tree_context(request)),

File "C:\Python35\Lib\json\__init__.py" in dumps
  230.         return _default_encoder.encode(obj)

File "C:\Python35\Lib\json\encoder.py" in encode
  198.         chunks = self.iterencode(o, _one_shot=True)

File "C:\Python35\Lib\json\encoder.py" in iterencode
  256.         return _iterencode(o, 0)

File "C:\Python35\Lib\json\encoder.py" in default
  179.         raise TypeError(repr(o) + " is not JSON serializable")

Exception Type: TypeError at /admin/catalogue/category/
Exception Value: UUID('659dd628-9f85-4a73-beb2-ba676ea4e2a2') is not JSON serializable
```

### Relevant bit

```
File "C:\Users\tomg\virtualenv\django\lib\site-packages\mptt\admin.py" in changelist_view
  199.                     'data-context': json.dumps(self._tree_context(request)),
```

**Comment by ad-m** [comment]
It looks python serializers should be replaced into django build-in
serializers which support uuid fields.

Wysłano z urządzenia mobilnego.

26.12.2016 1:39 AM "Tomasz Jan Góralczyk" <notifications@github.com>
napisał(a):

**Comment by matthiask** [comment]
Does the code in pull request #531 work for you?

**Comment by craigds** [comment]
I've just merged #531, assume it fixes this. If not we can reopen
